tests/unit_tests/export_onnx_test.py

```diff
@@ -15,6 +15,7 @@ def test_models_onnx_export_with_deprecated_input_shape(self):
         with tempfile.TemporaryDirectory() as tmpdirname:
             out_path = os.path.join(tmpdirname, "resnet18.onnx")
             models.convert_to_onnx(model=pretrained_model, out_path=out_path, input_shape=(3, 256, 256), pre_process=preprocess)
+            self.assertTrue(os.path.exists(out_path))
```

Comment:
I think you may want to use `os.path.isfile` here, since `os.path.exists` would return true if `out_path` would be a directory.

Reply:
We are using .exists() all over the codebase, nt on directories (see logging filenmes as an example).
I will fix that. Just letting you know it is everywhere and not a convention. 

If it is bugging us, let's fix all the occurenced in the codebase that are not directories and use .exists()